### PR TITLE
CSS and other improvements to Zen mode

### PR DESCRIPTION
**Pull request by pitaj**

### CODE REVIEW

File: static/lib/composer/formatting.js [hunk 1]
```diff
@@ -24,15 +24,18 @@ define('composer/formatting', ['composer/controls', 'composer/preview', 'compose
...
-					});
+					function onResize() {
+						if (!screenfull.isFullscreen) {
+							app.toggleNavbar(true);
+							$('html').removeClass('zen-mode');
+							resize.reposition(postContainer);
+							$(window).off('resize', onResize);
+						}
+					}
+					$(window).on('resize', onResize);
```
Comment (julianlam) [inline static/lib/composer/formatting.js]: I used `.one()` here instead of `.on()` specifically because I only wanted one event handler added when zen mode is opened. TIL if it's a named function, it doesn't get added twice. Neat.

File: static/templates/composer.tpl [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div component="composer" class="composer<!-- IF resizable --> resizable<!-- ENDIF resizable -->">
+<div component="composer" class="composer<!-- IF resizable --> resizable<!-- ENDIF resizable --><!-- IF isTopicOrMain --> topic-or-main<!-- ENDIF isTopicOrMain -->">
```
Comment (julianlam) [inline static/templates/composer.tpl]: What's this you're sneaking in here? :stuck_out_tongue:

Comment (pitaj) [inline static/templates/composer.tpl]: It's a class so you can hide things based on if the composer is composing a
main post.

Comment (julianlam) [inline static/templates/composer.tpl]: Remove and re-submit as separate PR please. There are two composer templates, and you have to ensure that `isTopicOrMain` is passed to the `/compose` route as well.

Comment (pitaj) [inline static/templates/composer.tpl]: The point of this change is to hide the title in zen mode when someone is simply replying to provide for even more space.

File: static/lib/composer/formatting.js (hunk 1 above)
Comment (pitaj) [inline static/lib/composer/formatting.js]: Notice line 35. I remove the handler. I changed it to do it this way because this way I can do nothing if it didn't actually leave fullscreen.

**Comment by julianlam** [comment]
Looks good :+1:
